CONTRIBUTING.md

```diff
@@ -130,16 +130,13 @@ For more details, please visit: [website]
 
 """
 
-from __future__ import absolute_import
-from __future__ import division
-from __future__ import print_function
 
 # -- import whatever you need here
 
-import mirdata.track as track
-import mirdata.utils as utils
 import mirdata.download_utils as download_utils
 import mirdata.jams_utils as jams_utils
+import mirdata.track as track
```

Comment:
while we're here - should we maybe standardize all these style imports to `from mirdata import track` and `from mirdata import utils`?

Reply:
Just saw you're using this convention below - let's change it here in the CONTRIBUTING too!

Reply:
Noted

Reply:
fixed in a7fe72e